### PR TITLE
Etcd elasticsearch lxd

### PR DESCRIPTION
**Pull request by mbruzek**
The etctd, elasticsearch charms needed to be updated to the latest revision number.

Also documented the LXD limitation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,9 +88,13 @@ ElasticSearch is used to hold all the log data and server information logged by
 
  The following issues still need to be resolved with this solution and are being worked on:
 
- - Killing the the Kubernetes leader will result in loss of cluster PKI.
- - No easy way to address the pods from the outside world
- - ZFS does not work on trusty (a bug that will be fixed soon) yet.
+ - This bundle is not supported on LXD because Juju needs to use a LXD profile
```
Comment (cmars) [inline README.md]: Thanks, this looks good.

File: README.md [hunk 2]
```diff
@@ -88,9 +88,13 @@ ElasticSearch is used to hold all the log data and server information logged by
...
+that can run Docker containers.
+ - Killing the the Kubernetes leader will result in loss of private key
+infrastructure (PKI).
+ - No easy way to address the pods from the outside world.
+ - The storage feature with ZFS does not work with trusty at this time because
+the code has to be enhanced to load the zfs module.
```
Comment (chuckbutler) [inline README.md]: *clack clack clack* enhance *clack clack clack* enhance

File: bundle.yaml [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
-    charm: "cs:~containers/trusty/etcd-2"
+    charm: cs:~containers/trusty/etcd-2
     num_units: 3
     annotations:
-      "gui-x": "1200"
-      "gui-y": "600"
-    constraints: "arch=amd64 cpu-cores=2 mem=4G root-disk=12G"
+      gui-x: '1200'
+      gui-y: '600'
+    constraints: arch=amd64 cpu-cores=2 mem=4G root-disk=12G
```
Comment (cmars) [inline bundle.yaml]: I'm not sure about the formatting changes in this file, but if it still works, then it's fine.

**Comment by chuckbutler** [comment]
+1 LGTM

**Comment by cmars** [comment]
LGTM

File: bundle.yaml (hunk 3 above)
Comment (mbruzek) [inline bundle.yaml]: I built a tool to query the charm store to update the revno of the charms. The round trip dumps out different quotes.

Comment (mbruzek) [inline bundle.yaml]: It is valid YAML, whether Juju will work with it remains to be seen.

**Comment by chuckbutler** [comment]
Not sure how this happened, but the branch has conflicts now :octopus:

**Comment by mbruzek** [comment]
This branch has conflicts because of other changes landing before it. I am going to close this pull request and fix the code in another branch.

**Pull request closed without merging**